### PR TITLE
Cockpit 325

### PR DESCRIPTION
**Pull request by martinpitt**
Preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-325.html

### CODE REVIEW

File: _posts/2024-09-25-cockpit-325.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 325 and cockpit-machines 320:
+
+
+## Storage: Support for managing virtual Stratis filesystem sizes
+
+Cockpit now directly exposes the `--size` and `--size-limit` options of the `stratis filesystem create` command as well as the `stratis filesystem set-size-limit` and `stratis filesystem unset-size-limit` commands.
```
[the remark this answers is not in the document]
Comment (martinpitt) [reply _posts/2024-09-25-cockpit-325.md]: Obsolete, blog for this feature punted to the next release.

File: _posts/2024-09-25-cockpit-325.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+![screenshot of support for managing virtual stratis filesystem sizes]({{ site.baseurl }}/images/325-storage-support-for-managing-virtual-stratis-filesystem-sizes.png)
```
[the remark this answers is not in the document]
Comment (martinpitt) [reply _posts/2024-09-25-cockpit-325.md]: Obsolete, blog for this feature punted to the next release.

File: _posts/2024-09-25-cockpit-325.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 325 and cockpit-machines 320:
+
+## Machines: Improvements to snapshots of running machines
+
+The snapshot of a running machine includes its memory state, and Cockpit sometimes asks where to store it. This is now done in a slightly improved fashion: you only need to specify the directory, and Cockpit will provide the name of the file. The default for this directory now also takes previous snapshots into account, so that you don't need to touch it if you always store the memory state of all snapshots in the same directory. Plus, Cockpit now gives estimates for the amount of space that will be needed and warns you if there doesn't seem to be enough.
```
Comment (garrett) [inline _posts/2024-09-25-cockpit-325.md:18]: There are a lot of features; what if we made it a list?

```suggestion
## Machines: Snapshot improvements

Snapshotting running machines has improved in several ways:

- Snapshots now include memory state
- Only a directory needs to be specified; Cockpit will generate a filename
- The snapshot dialog remembers the storage path, making future snapshots easier
- Space estimates are shown, including a warning if there isn't enough storage available
```

**Review by garrett — CHANGES_REQUESTED** [review]
The paragraph felt a bit long, with several different features, so I thought splitting it into a bulleted list makes sense. What do you think?

File: _posts/2024-09-25-cockpit-325.md (hunk 3 above)
Comment (martinpitt) [reply _posts/2024-09-25-cockpit-325.md]: > Snapshots now include memory state

That is not a new feature, that is just a description of the existing feature. Hence, I changed it to a list, but with adjusted wording.

**Review by garrett — APPROVED** [review]
This works. Thanks!

**Pull request merged**